src/SkyWalking.AspNetCore/SkyWalking.AspNetCore.csproj

```diff
@@ -15,7 +15,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Hosting" Version="2.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Abstractions" Version="2.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DiagnosticAdapter" Version="2.1.0-preview2-final" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="2.1.0-preview2-final" />
```

Comment:
can we changes the package version to 2.0.0 ?

Reply:
`Microsoft.Extensions.DiagnosticAdapter` should be removed.

Reply:
package `Microsoft.Extensions.Http` and `Microsoft.Extensions.Logging.Abstractions` and `Microsoft.Extensions.Options` versions , Maybe it's not part of this submission.

Reply:
Refactoring HttpClientFactory should be in a new PR.

Reply:
These submissions are only refactoring `SkyWalking.Diagnostics`.

Reply:
Agree !